locale/en_US/LC_MESSAGES/django.po

```diff
@@ -1668,8 +1668,8 @@ msgstr "Please go to the following page and setup your account:"
 msgid "Please join the SEED-Platform Dev forum where you can connect with other developers."
 msgstr "Please join the SEED-Platform Dev forum where you can connect with other developers."
 
-msgid "Please review SEED headers.  Duplicate values are not allowed."
-msgstr "Please review SEED headers.  Duplicate values are not allowed."
+msgid "Please review SEED headers or Data File headers. Duplicate values are not allowed for either."
```

Comment:
Can we reword to 'Please review SEED column names or the headers of the file being imported. Duplicate values are not allowed in either case.'